src/config/partners/mirror-wellness.ts

```diff
@@ -1,11 +1,11 @@
 import { Partner } from './types'
-import { limited2 } from '../test-contracts'
 import { CAMPAIGN_HOUR, CAMPAIGN_MINUTE } from '@/config/constants'
+import { MintType } from '@/components/MintDialog/types'
 
 const mirrorWellness: Partner = {
   slug: 'mirrorswellnessclub',
   name: 'Mirrors Wellness Club',
-  url: 'https://www.mirror.co/',
+  url: 'http://mirrorswellnessclub.io',
```

Comment:
Was this an accident?

```suggestion
  url: 'https://mirrorswellnessclub.io',
```